spec/rubocop/cop/rails/where_missing_spec.rb

```diff
@@ -0,0 +1,207 @@
+      expect_offense(<<~RUBY)
+        Foo.left_joins(:foo).where(foos: { id: nil }, bar: "bar")
+            ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        Foo.where.missing(:foo).where(bar: "bar")
+      RUBY
+    end
+
+    it "registers an offense when using `left_joins(foo).where(bar: 'bar', foos: {id: nil})`" do
+      expect_offense(<<~RUBY)
+        Foo.left_joins(:foo).where(bar: "bar", foos: { id: nil })
+            ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        Foo.where.missing(:foo).where(bar: "bar")
+      RUBY
+    end
+
+    it 'registers an offense when using `where(foos: {id: nil}).joins(:bar).left_joins(foo)`' do
+      expect_offense(<<~RUBY)
+        Foo.left_joins(:foo).joins(:bar).where(foos: { id: nil })
+            ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        Foo.where.missing(:foo).joins(:bar)
+      RUBY
+    end
+
+    it 'registers an offense when using `left_joins(foo).where(foos: {id: nil})` with multi-line leading ' \
+       'dot method calls' do
+      expect_offense(<<~RUBY)
+        Foo
+          .left_joins(:foo)
+           ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
+          .where(foos: { id: nil })
+          .where(bar: "bar")
+      RUBY
+
+      expect_correction(<<~RUBY)
+        Foo
+          .where.missing(:foo)
+          .where(bar: "bar")
+      RUBY
+    end
+
+    it 'registers an offense when using `left_joins(foo).where(foos: {id: nil})` with multi-line trailing ' \
+       'dot method calls' do
+      expect_offense(<<~RUBY)
+        Foo.
+          left_joins(:foo).
+          ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
+          where(foos: { id: nil }).
+          where(bar: "bar")
+      RUBY
+
+      expect_correction(<<~RUBY)
+        Foo.
+          where.missing(:foo).
+          where(bar: "bar")
+      RUBY
+    end
+
+    it 'registers an offense when using `left_joins(foo).where(foos: {id: nil})` and there is a line break after' \
+       '`left_joins.where`' do
+      expect_offense(<<~RUBY)
+        Foo.left_joins(:foo).where(foos: { id: nil })
+            ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
+          .where(bar: "bar")
+      RUBY
+
+      expect_correction(<<~RUBY)
+        Foo.where.missing(:foo)
+          .where(bar: "bar")
+      RUBY
+    end
+
+    it 'registers an offense when using `left_joins(foo).where(foos: {id: nil})` and there is a line break after' \
+       '`left_joins.where` and receiver' do
+      expect_offense(<<~RUBY)
+        Foo
```

Comment:
Again for spec completeness
```
Foo
           .left_joins(:foo).where(bar: "bar")
           .where(foos: { id: nil })
```

Reply:
I think it looks good. Thank you so much. I added it here.
https://github.com/rubocop/rubocop-rails/blob/fe2ecf8a8a3fa5975e7201e066402a9ec5f690a9/spec/rubocop/cop/rails/where_missing_spec.rb#L178-L191